tests/foreman/ui/test_contenthost.py

```diff
@@ -1798,7 +1798,7 @@ def test_search_for_virt_who_hypervisors(session, default_location):
 @pytest.mark.destructive
 @pytest.mark.run_in_one_thread
 @pytest.mark.upgrade
-def test_content_access_after_stopped_foreman(foreman_service_teardown, rhel7_contenthost):
```

Comment:
In addition to this test, there is another test[1] added that is similar, which might be worth considering for this target_sat simplification. Therefore, you could completely remove this `foreman_service_teardown` 

And `:CaseComponent: Infrastructure` is owned by @lpramuk , so I think `:Assignee:` shouldn't be removed
 
[1] https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/sys/test_foreman_service.py

Reply:
@Gauravtalreja1 I am a bit confused, "foreman_service_teardown" has been removed, or should I say it is removed in this PR.

Re your last point: If @lpramuk wants this test he can raise a PR to move it to his module. I think its unfair on a component owner to have other another component owner's test in their module. I will remove the CaseComponent.